leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -138,7 +138,10 @@ <V extends Persistable & Versionable> boolean[] writeBatchOfRounds(
                 @Bind("useCase") String useCase,
                 @BindPojo("round") Iterable<PaxosRound<V>> rounds);
 
-        @SqlQuery("SELECT DISTINCT(namespace) FROM paxosLog")
-        Set<String> getAllNamespaces();
+        @SqlQuery("SELECT MIN(namespace) FROM paxosLog")
+        Optional<String> getSmallestNamespace();
```

Comment:
Two queries, resisted the temptation of removing this and using `""` for the first query

I think this is fine